packages/boot/tools/supports.ts

```diff
@@ -476,9 +507,13 @@ export const makeSwingsetTestKit = async (
       case `${BridgeId.DIBC}:IBC_METHOD`:
       case `${BridgeId.VTRANSFER}:IBC_METHOD`: {
         switch (obj.method) {
-          case 'startChannelOpenInit':
-            pushInbound(BridgeId.DIBC, icaMocks.channelOpenAck(obj));
+          case 'startChannelOpenInit': {
+            const message = icaMocks.channelOpenAck(obj, bech32Prefix);
+            shouldAckImmediately(bridgeId, 'startChannelOpenInit')
+              ? inbound(BridgeId.DIBC, message)
+              : pushInbound(BridgeId.DIBC, message);
```

Comment:
ternaries are for expressions, not control flow.
```suggestion
            const handle = shouldAckImmediately(bridgeId, 'startChannelOpenInit') ? inbound : pushInbound;
            handle(BridgeId.DIBC, message);
```